### ISSUE TITLE
Unittests failing

### ISSUE THREAD

**Issue by rikkimax** [p0nce/commonmark-d#3]
Looks like the tests are not being ran recently:

```
649 tests parsed.
Test 10: FAIL
***Markdown:

#	Foo


*** Expected (length 12):

<h1>Foo</h1>

*** Got instead (length 13):

<h1>	Foo</h1>

See specifications lines 466-470 section Tabs
Test 82: FAIL
***Markdown:

  Foo *bar
baz*	
====


*** Expected (length 28):

<h1>Foo <em>barbaz</em></h1>

*** Got instead (length 29):

<h1>Foo <em>barbaz</em>	</h1>

See specifications lines 1378-1385 section Setext headings
647 / 649 tests passed.
```

I had to also manually copy the ``spec-tests.json`` file, otherwise tests will fail when testing the depender.

**Comment by p0nce** [comment]
I don't think they have ever passed 100%, I think it's the same 2 unittests since the beginning that are failing. (that said, would be cool to fix them)

**Comment by p0nce** [comment]
Fixed in 1.0.10

**Comment by rikkimax** [comment]
Thanks!
